### PR TITLE
bug 1224284 - Crontabber app to sync featured versions

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

**Comment by peterbe** [comment]
ping @twobraids !
I'd like to get this up and running stage.

**Comment by peterbe** [comment]
@adngdb I assigned this review to @twobraids before the new order. I guess you'll have to get yourself deep into crontabbery stuff now :)

File: socorro/unittest/cron/jobs/test_featured_versions_sync.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+import mock
+from nose.tools import eq_
+
+from crontabber.app import CronTabber
+from socorro.unittest.cron.setup_configman import (
+    get_config_manager_for_crontabber,
+)
+
+
```
Comment (adngdb) [inline socorro/unittest/cron/jobs/test_featured_versions_sync.py]: Useless newline.

**Comment by adngdb** [comment]
r+

**Pull request merged**